start.xenit.eu/src/main/java/eu/xenit/alfred/initializr/start/build/SimpleBuildCustomizer.java

```diff
@@ -27,9 +30,11 @@
  */
 public class SimpleBuildCustomizer implements BuildCustomizer<Build> {
 
+    private static final Log logger = LogFactory.getLog(SimpleBuildCustomizer.class);
```

Comment:
Since we're using Lombok, can we use logging annotations like `@Slf4j`?

Reply:
Or remove because we're not logging anything (anymore) ? :-)